tests/installation/install_and_reboot.pm

```diff
@@ -48,6 +48,7 @@ sub run {
     my $self = shift;
     # NET isos are slow to install
     my $timeout = 2000;
+    $timeout *= 2 if check_var('ARCH', 'aarch64');
```

Comment:
Needs conflict resolution. Also that should deserve a comment.

Reply:
There, just added the conflict resolution, and the comment. However I'm not so happy because we might just end up waiting 2.2 hrs in a very bad scenario :)

```
    # aarch64 can be particularily slow depending on the hardware
    $timeout *= 2 if check_var('ARCH', 'aarch64') && get_var('MAX_JOB_TIME');
    # encryption, LVM and RAID makes it even slower
    $timeout *= 2 if (get_var('ENCRYPT') || get_var('LVM') || get_var('RAID'));
```